src/SearchField/SearchFieldInput.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Input from '../Input';
+import { SearchFieldContext } from './SearchFieldAdvanced';
+
+const SearchFieldInput = (props) => {
+  const { className, ...others } = props;
+
+  const {
+    inputId, value, handleChange, handleFocus, handleBlur, refs,
+  } = useContext(SearchFieldContext);
+
+  return (
+    <Input
+      {...others}
+      ref={refs.input}
+      type="search"
```

Comment:
Should we bother using `type="search"` instead of `type="text"` if it just causes styling problems? cc @wittjeff

Reply:
That's a fair point. As far as I know, using `type="search"` doesn't add any functionality different than using `type="text"`; it only adds stylistic differences, which we're overriding anyways.

I'm going to switch to using `type="text"` unless @wittjeff says otherwise.